tests/src/test/scala/org/http4s/MessageSpec.scala

```diff
@@ -126,6 +126,40 @@ class MessageSpec extends Http4sSpec {
       }
     }
 
+    "cookies" should {
+      val cookieList = List(
+        RequestCookie("test1", "value1"),
+        RequestCookie("test2", "value2"),
+        RequestCookie("test3", "value3"))
+
+      "be empty if there are no Cookie headers present" in {
+        Request(Method.GET).cookies mustEqual List.empty
+      }
+
+      "parse discrete HTTP/1 Cookie header(s) into corresponding RequestCookies" in {
+        val cookies = Header("Cookie", "test1=value1; test2=value2; test3=value3")
+        val request = Request(Method.GET, headers = Headers.of(cookies))
```

Comment:
Does your test require specifying the HTTP version when constructing the [Request](https://github.com/http4s/http4s/blob/master/core/src/main/scala/org/http4s/Message.scala#L242)?

Reply:
No. That should not impact the way the headers are handled. If I need to add that for semantics, I can, but this specifically refers to the way the cookies are being sent.

Reply:
👍